app/src/sharedTest/java/org/oppia/android/app/faq/FAQListFragmentTest.kt

```diff
@@ -98,7 +98,8 @@ class FAQListFragmentTest {
       )
       onView(
         atPositionOnView(
-          R.id.faq_fragment_recycler_view, 0, R.id.faq_question_text_view
+          recyclerViewId = R.id.faq_fragment_recycler_view,
+          position = 0, R.id.faq_question_text_view
```

Comment:
In this the third argument is not a named argument `R.id.faq_question_text_view`

Reply:
@ksog66 Can you reply this comment so that I can know whether you applied these changes or faced any issues.
You can write as simple as "Done".

Reply:
@rt4914 Done